contexts/PaymentContext/src/Domain/CalculatingPaymentDelayCalculator.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\PaymentContext\Domain;
+
+use DateInterval;
+use DateTime;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ */
+class CalculatingPaymentDelayCalculator implements PaymentDelayCalculator {
```

Comment:
More seriously though, `Default` or `Normal` are probably better prefixes, and perhaps the domain has some better terms to offer. Question to ask: how would you identify this type of payment calculation when talking to the fun people?

If this is the only implementation we have, it is general, and you don't expect more, then finding a good name is kinda weird since the most obvious one is already stolen by the interface. And in such a case it is not clear to me you need the interface. (Not saying you should not have it)